phpdotnet/phd/Package/PHP/Web.php

```diff
@@ -211,6 +223,40 @@ protected function writeJsonIndex() {
         v("Index written", VERBOSE_FORMAT_RENDERING);
     }
 
+    protected static function generateDeprecatedInfo($filename) {
+        static $info;
```

Comment:
(and maybe make an inline note of the different str_replace() - thats not easy to spot and someone could accidentally merge these functions without noticing)